components/esp_lvgl_port/CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## 2.6.0
 
-### Features
+## Features
```

Comment:
Nitpick: Why was this changed to `##`?

Reply:
I am sorry, I mistakenly removed it during merge. Thanks for notice.